src/javascript/binary/pages/trade/common.js

```diff
@@ -741,13 +741,14 @@ function marketOrder(market){
         france: 9,
         belgium: 10,
         amsterdam: 11,
-        commodities: 12,
-        metals: 13,
-        energy: 14,
-        volidx: 15,
-        random_index: 16,
-        random_daily: 17,
-        random_nightly: 18
+        otc_stock: 12,
```

Comment:
I can only see `otc_stock` in the list not `otc_index`. Should I add it too?

For `otc_stock`: it's there because it is a submarket of Stocks market. If it is not listed here means order = `0` and zero is before france (9), belgium (10) and amsterdam (11), so it is listed the first item in the submarkets list of Stocks.